code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -171,6 +174,8 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		/obj/item/roller = 1,
 		/obj/item/spear = 1,
 		/obj/item/weldingtool/largetank = 1,
+		/obj/item/gun/ballistic/automatic/pistol = 1,
```

Comment:
```suggestion
		/obj/item/gun/ballistic/automatic/pistol/divet/rb = 1,
```
Pistols should be divet variation for now, until I can properly look at all the guns.